courses-app/src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,49 @@
+import Button from '../../../../common/Button/Button';
+import { SHOW_COURSE_BTN_TEXT } from '../../../../constants';
+import getCourseDuration from '../../../../helpers/getCourseDuration';
+import getDatePoint from '../../../../helpers/getDatePoint';
+
+import './CourseCard.css';
+
+function CoursesCard(props) {
+	const { title, description, creationDate, duration, authorsNameList } = {
+		...props,
+	};
```

Comment:
Can be simplified

const { title, description, creationDate, duration, authorsNameList } = props;